src/hotspot/share/code/codeHeapState.cpp

```diff
@@ -1331,7 +1331,7 @@ void CodeHeapState::print_usedSpace(outputStream* out, CodeHeap* heap) {
 
   if (SizeDistributionArray != nullptr) {
     unsigned long total_count = 0;
-    unsigned long total_size  = 0;
+    uint64_t total_size  = 0;
```

Comment:
This is a functional change (actually the only one I could spot in this whole PR), and it increases the size of this type to 64-bit. 

Its also incorrect, since on 32-bit platforms you probably don't want 64-bit here. The correct type to use would be size_t.

Since this change is hidden in a deluge of unnecessary cosmetic changes, if we feel this is needed, this should be addressed in a separate RFE. 

But then again, probably not: code cache size is limited to 32-bit, and I bet we have a lot more places to change if we wanted to change that.

Also way out of scope for the RFE description.

Reply:
For this one (and the one below) I couldn't figure out whether the intention was for a 32 or 64 bit number in this case. This would be 64 bit on every platform but on Windows, which is 32 bit, and the code directly below this also faces the same issue too, is this intentional?

Reply:
long is 32-bit on Linux x86 and arm32 too.

Reply:
[not a review, just a drive-by comment]
The type `long` really shouldn't be used in shared code.  There have been discussions and maybe some efforts
to nuke uses, but having just done the grep, there are still quite a few.  The one in question might actually be an
appropriate place to use `uintx`.

Reply:
Hi Kim, mind if you share the grep command for that?